apis/v1alpha2/nginxproxy_types.go

```diff
@@ -365,30 +365,27 @@ type NginxPlus struct {
 // Corresponds to the NGINX resolver directive: https://nginx.org/en/docs/http/ngx_http_core_module.html#resolver
 type DNSResolver struct {
 	// Timeout specifies the timeout for name resolution.
-	// Default: 30s.
 	//
 	// +optional
 	Timeout *v1alpha1.Duration `json:"timeout,omitempty"`
 
 	// CacheTTL specifies how long to cache DNS responses.
-	// Default: 30s.
 	//
 	// +optional
 	CacheTTL *v1alpha1.Duration `json:"cacheTTL,omitempty"`
 
-	// IPv6 enables IPv6 lookups.
-	// Default: true.
+	// DisableIPv6 disables DisableIPv6 lookups.
+	// If not specified, or set to false, IPv6 lookups will be enabled.
 	//
 	// +optional
-	IPv6 *bool `json:"ipv6,omitempty"`
+	DisableIPv6 *bool `json:"ipv6,omitempty"`
 
 	// Addresses specifies the list of DNS server addresses.
 	// Each address can be an IP address or hostname.
 	// Example: [{"type": "IPAddress", "value": "8.8.8.8"}, {"type": "Hostname", "value": "dns.google"}]
 	//
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=16
-	// +listType=set
```

Comment:
Why remove this? can we not ensure unique items in this list?

Reply:
No, it caused errors - see [the failed run here](https://github.com/nginx/nginx-gateway-fabric/actions/runs/17075028525/job/48414611343)

From what I can gather, this would only work on something simple like `[]string`